js/client.js

```diff
@@ -753,11 +754,15 @@ function toId() {
 						// anyway, this affects SockJS because it makes HTTP requests to localhost
 						// but it turns out that making direct WebSocket connections to localhost is
 						// still supported, so we'll just bypass SockJS and use WebSocket directly.
+						var possiblePort = new URL(document.location + self.query).searchParams.get('port');
+						// We need to bypass the port as well because on most modern browsers, http gets forced
+						// to https, which means a ws connection is made to port 443 instead of wherever it's actually running,
+						// thus ensuring a failed connection.
+						var port = possiblePort ? possiblePort[1] : Config.server.port;
```

Comment:
I think you forgot to update this after the searchParams change

Reply:
Yeah. 
```suggestion
						var port = possiblePort || Config.server.port;
```